resume-buddy/src/main/java/com/google/sps/servlets/DeleteBlobServlet.java

```diff
@@ -0,0 +1,37 @@
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/delete-blob")
+public class DeleteBlobServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    boolean isBlobDeleted = false;
+
+    String resumeBlobKey = request.getParameter("resumeBlobKey");
+    BlobKey blobKey = new BlobKey(resumeBlobKey);
+    blobstoreService.delete(blobKey);
+
+    // check if blob is empty (not sure if this will work, I dont know what the blobkey is being set
+    // too after its deleted)
+    if (blobKey == null) {
+      isBlobDeleted = true;
+    }
```

Comment:
Looks like the way to check if the blob no longer exists is to use [`BlobInfoFactory.loadBlobInfo`](https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/blobstore/BlobInfoFactory.html#loadblobinfo). If the return value is null, the blob no longer exists.

``` 
BlobInfoFactory blobInfoFactory = new BlobInfoFactory();
isBlobDeleted = blobInfoFactory.loadBlobInfo(blobKey) == null;
```